### PR TITLE
Add burn/destroy/melt commands

### LINKED ISSUE #100: Add burn/melt/destroy commands

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add burn/destroy/melt commands

## Links to any relevant issues

Fixes #100 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

By calling the commands

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/03_account.md [hunk 1]
```diff
@@ -27,6 +27,27 @@ Prints the account balance.
...
+### `burn-nft`
+
+Burns an NFT.
+
+#### Parameters
+
+| Name     | Optional  | Default | Example    |
+| -------- | --------- | ------- | ---------- |
+| `nft_id` | ✘         | N/A     | "0xabcdef" |
+
```
Comment (thibault-martinez) [inline documentation/docs/03_account.md:64]: Can you provide examples like the other commands please ?

File: src/account.rs [hunk 2]
```diff
@@ -56,14 +57,25 @@ pub async fn account_prompt_internal(account_handle: AccountHandle) -> Result<bo
...
+                AccountCommand::DestroyFoundry { foundry_id } => {
+                    destroy_foundry_command(&account_handle, foundry_id).await
+                }
                 AccountCommand::Exit => {
                     return Ok(true);
                 }
                 AccountCommand::Faucet { url, address } => faucet_command(&account_handle, url, address).await,
                 AccountCommand::Addresses => addresses_command(&account_handle).await,
                 AccountCommand::Transactions => transactions_command(&account_handle).await,
+                AccountCommand::MeltNativeToken { token_id, amount } => {
```
Comment (thibault-martinez) [inline src/account.rs:74]: I know it's not part of your PR but can you reorder them alphabetically please ? Looks like I forgot to do it when I renamed the variants

File: documentation/docs/03_account.md [hunk 3]
```diff
@@ -27,6 +27,41 @@ Prints the account balance.
...
 
+### `burn-native-token`
+
+Burns a native token.
+
+#### Parameters
+
+| Name       | Optional  | Default | Example    |
+| ---------- | --------- | ------- | ---------- |
+| `token_id` | ✘         | N/A     | "0xabcdef" |
```
Comment (thibault-martinez) [inline documentation/docs/03_account.md]: Nitpicking but this example is wrong, let's not give incorrect info, just reuse the one from below

File: documentation/docs/03_account.md [hunk 4]
```diff
@@ -27,6 +27,41 @@ Prints the account balance.
...
+
+### `burn-nft`
+
+Burns an NFT.
+
+#### Parameters
+
+| Name     | Optional  | Default | Example    |
+| -------- | --------- | ------- | ---------- |
+| `nft_id` | ✘         | N/A     | "0xabcdef" |
```
Comment (thibault-martinez) [inline documentation/docs/03_account.md]: same and for following

File: documentation/docs/03_account.md (hunk 3 above)
Comment (Thoralf-M) [reply documentation/docs/03_account.md]: You started with that :carlos:

Comment (thibault-martinez) [reply documentation/docs/03_account.md]: I just did for foundry metadata but there it's a correct value

File: documentation/docs/03_account.md [hunk 5]
```diff
@@ -27,6 +27,41 @@ Prints the account balance.
...
+
+| Name     | Optional  | Default | Example                                                              |
+| -------- | --------- | ------- | -------------------------------------------------------------------- |
+| `nft_id` | ✘         | N/A     | "0x397ae8552dcf0dc604a44c9d86a5005d09f95d67e2965ea3b1c1271f9a9ae44c" |
+
+#### Example
+
+Burn a provided NFT.
+```sh
+> Account "main": burnt-nft 0x397ae8552dcf0dc604a44c9d86a5005d09f95d67e2965ea3b1c1271f9a9ae44c
```
Comment (thibault-martinez) [inline documentation/docs/03_account.md]:
```suggestion
> Account "main": burn-nft 0x397ae8552dcf0dc604a44c9d86a5005d09f95d67e2965ea3b1c1271f9a9ae44c
```

File: documentation/docs/03_account.md [hunk 6]
```diff
@@ -59,6 +94,38 @@ Note that only Basic Outputs with only an address unlock condition can be consol
...
+#### Parameters
+
+| Name         | Optional  | Default | Example                                                                          |
+| ------------ | --------- | ------- | -------------------------------------------------------------------------------- |
+| `foundry_id` | ✘         | N/A     | "0x08b2bcba092bfb3fe3a12afcf21115016b27d833a7c456404fe2fe0921799f24dd0100000000" |
+
+#### Example
+
+```sh
+> Account "main": destroy-alias 0x08b2bcba092bfb3fe3a12afcf21115016b27d833a7c456404fe2fe0921799f24dd0100000000
```
Comment (thibault-martinez) [inline documentation/docs/03_account.md]:
```suggestion
> Account "main": destroy-foundry 0x08b2bcba092bfb3fe3a12afcf21115016b27d833a7c456404fe2fe0921799f24dd0100000000
```

File: src/command/account.rs [hunk 7]
```diff
@@ -95,6 +105,40 @@ pub async fn addresses_command(account_handle: &AccountHandle) -> Result<(), Err
     Ok(())
 }
 
+// `burn-native-token` command
+pub async fn burn_native_token_command(
+    account_handle: &AccountHandle,
+    token_id: String,
+    amount: String,
+) -> Result<(), Error> {
+    log::info!("Burning native token {amount} {token_id}.");
```
Comment (thibault-martinez) [inline src/command/account.rs]: Reverse for consistency

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**